File: codes_python/A1-040.py
cal = {1:100, 2:120, 3:200, 4:60}
total = 0
while True:
    choice = int(input())
    if choice == 5:
        print("Bye Bye")
        print(f"Total Calories: {total}")
        break
    total += cal.get(choice, 0)

File: codes_c/A1-040.c
#include <stdio.h>
int main()
{
    int choice, total = 0;
    while (scanf("%d", &choice) == 1)
    {
        if (choice == 5)
            break;
        if (choice == 1)
            total += 100;
        else if (choice == 2)
            total += 120;
        else if (choice == 3)
            total += 200;
        else if (choice == 4)
            total += 60;
    }
    printf("Bye Bye\nTotal Calories: %d\n", total);
    return 0;
}
